README.md

```diff
@@ -142,15 +146,19 @@ If Swagger format is configured, [Swagger UI](https://swagger.io/tools/swagger-u
 You can add this configurations to your `config/test.ex`
 
 -   information_source: the module with doc information
--   output_file: a custom name to the output file
--   doc_format: ApiBlueprint or Swagger formats
+-   output: a custom name to the output file
+-   format: ApiBlueprint or Swagger formats
 -   env_var: a custom name to the env to active XCribe.Formatter
+-   json_library: The library to be used for json decode/encode
 
 Example
 
 ```elixir
-config :xcribe, :information_source, YourAppWeb.Information
-config :xcribe, :output_file, "API-DOCUMENTATION.apib"
-config :xcribe, :doc_format, :swagger # or :api_blueprint
-config :xcribe, :env_var, "DOC_API"
+config :xcribe, :configuration, [
+  information_source: YourAppWeb.Information,
+  output: "API-DOCUMENTATION.apib",
+  format: :swagger # or :api_blueprint,
+  env_var: "DOC_API",
+  json_library: Jason
```

Comment:
Only `Jason` and `Poison` are supported, right? I think that we should add this information explicit here to avoid developers trying with other libs.

Reply:
Any library that implements the same API for `encode/2`, `encode/2!`, `decode/2` and `decode!/2` can be used. Any way I added to Readme a link for `Xcribe.JSON` where there is the info about `Json` and `Poison`.